src/posts/cybersecurity-risks.md

```diff
@@ -27,31 +27,8 @@ or use [WiFi to see humans through walls](https://www.marktechpost.com/2023/02/1
 There will most likely come a point where an AI is better at hacking than the best human hackers.
 This can go wrong in many ways.
 
-- **Nuclear**: Cyberweapons can be used to gain access to nuclear materials, meltdown a reactor, convince humans they need to fire nuclear missiles, or even fire them automatically.
```

Comment:
This deserves a source, but it should not be removed in its entirety. I remember reading some articles about it .

- https://www.nytimes.com/2017/03/14/opinion/why-our-nuclear-weapons-can-be-hacked.html
- https://www.nti.org/analysis/articles/cyber/

Deserves more discussion probably